Resources/config/phpcr-sitemap.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+
+<container
+        xmlns="http://symfony.com/schema/dic/services"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="cmf_seo.sitemap.phpcr_provider.class">Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SitemapUrlInformationProvider</parameter>
+    </parameters>
+
+    <services>
+        <service id="cmf_seo.sitemap.phpcr_provider" class="%cmf_seo.sitemap.phpcr_provider.class%">
```

Comment:
lets make this service private. it is always loaded even when not needed. and there is no point getting it from the container.